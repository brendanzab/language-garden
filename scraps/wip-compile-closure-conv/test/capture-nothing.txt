let a : Int := 1;
let f : Int -> Int :=
  fun (x : Int) => x;

f 100
