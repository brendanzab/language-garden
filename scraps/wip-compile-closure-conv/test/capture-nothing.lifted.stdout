def f0↑ (env1 : ()) (x2 : Int) := x2;
let a0 : Int := 1;
f0↑ () 100
